src/network-interface.js

```diff
@@ -4,7 +4,10 @@ import { extractRequestFiles } from './helpers'
 export class HTTPUploadNetworkInterface extends HTTPFetchNetworkInterface {
   fetchFromRemoteEndpoint({ request, options }) {
     // Skip upload proccess if SSR
-    if (typeof window !== 'undefined') {
+    if (
+      typeof window !== 'undefined' ||
+      (typeof global !== 'undefined' && typeof global.FormData !== 'undefined')
+    ) {
```

Comment:
Interesting idea to just look for the existence of FormData in global scope. Maybe we could replace the whole condition with just `if (FormData) {`